### ISSUE TITLE
Change StorageLocation element from StorageHierarchyScopeType to ResourceLocationType in MaterialLotType and MaterialSubLotType

### ISSUE THREAD

**Issue by racheliurui** [ETPartners/b2mmlv7beta#23]
https://github.com/MESAInternational/B2MML-BatchML/issues/30

### Background and Solution
to align with updated 950002 Operational Location Model in Material Model, the following changes are required.

complexType name = "MaterialLotType", element name ="StorageLocation"; 
CHANGE: type = "StorageHierarchyScopeType" to "ResourceLocationType"

complexType name = "MaterialSubLotType", element name ="StorageLocation"; 
CHANGE:  type = "StorageHierarchyScopeType" to "ResourceLocationType" 

### Impacted Types
_Material.xsd_
- MaterialLotType
- MaterialSubLotType

### Supporting Document
_950002 JWG5 CDV01 version (2019 12)_
Clause 5.7.6 Material lot, Table 69 – Material lot attributes
Clause 5.7.8 Material sublot, Table 73 – Material sublot attributes

**Comment by Dennis-Brandl** [comment]
I agree with the change (except to move of the OperationalLocation.xsd to common.)
